### PR TITLE
PBI #145531: Replace babel react-app preset

### PR DESCRIPTION
**Pull request by Kashkovsky**

### CODE REVIEW

File: .babelrc [hunk 1]
```diff
@@ -2,15 +2,21 @@
 	"presets": [
 		"@babel/env",
 		"@babel/typescript",
```
Comment (dmarcautan) [inline .babelrc]: Isn't it named @babel/preset-typescript?

File: package.json [hunk 2]
```diff
@@ -46,16 +46,18 @@
 		"xmldom": "^0.1.27"
 	},
 	"peerDependencies": {
-		"react": "^16.11.0",
```
Comment (dmarcautan) [inline package.json]: Why did you downgrade React? Also, Typescript version should be bumped to the latest

File: .babelrc (hunk 1 above)
Comment (Kashkovsky) [reply .babelrc]: Renamed for consistency

File: package.json (hunk 2 above)
Comment (Kashkovsky) [reply package.json]: Reverted since Elements has 16.11.0 now

**Pull request merged**